CMakeLists.txt

```diff
@@ -18,4 +18,5 @@ if (DSA_INCLUDE_TEST)
   add_subdirectory("${CMAKE_CURRENT_LIST_DIR}/benchmark")
   add_subdirectory("${CMAKE_CURRENT_LIST_DIR}/example")
   add_subdirectory("${CMAKE_CURRENT_LIST_DIR}/playground")
+  add_subdirectory("${CMAKE_CURRENT_LIST_DIR}/wss.0213")
```

Comment:
seems that wss.0123 is missing?

Reply:
Its a local directory. I will remove it.